### PR TITLE
drop colossalai from testing as no stable release yet

### PR DESCRIPTION
**Pull request by Borda**
## What does this PR do?

seems that they nuked all releases and left only future RC which is now already a month old
https://pypi.org/project/colossalai/#history
Thre is no point in testing RC and failing tests because of it...

## Before submitting

- [ ] Was this **discussed/approved** via a GitHub issue? (not for typos and docs)
- [x] Did you read the [contributor guideline](https://github.com/Lightning-AI/lightning/blob/master/.github/CONTRIBUTING.md), **Pull Request** section?
- [x] Did you make sure your **PR does only one thing**, instead of bundling different changes together?
- [ ] Did you make sure to **update the documentation** with your changes? (if necessary)
- [ ] Did you write any **new necessary tests**? (not for typos and docs)
- [ ] Did you verify new and **existing tests pass** locally with your changes?
- [ ] Did you list all the **breaking changes** introduced by this pull request?
- [ ] Did you **update the CHANGELOG**? (not for typos, docs, test updates, or minor internal changes/refactors)

## PR review

Anyone in the community is welcome to review the PR.
Before you start reviewing, make sure you have read the [review guidelines](https://github.com/Lightning-AI/lightning/wiki/Review-guidelines). In short, see the following bullet-list:

- [x] Is this pull request ready for review? (if not, please submit in draft mode)
- [x] Check that all items from **Before submitting** are resolved
- [x] Make sure the title is self-explanatory and the description concisely explains the PR
- [x] Add labels and milestones (and optionally projects) to the PR so it can be classified

## Did you have fun?

Make sure you had fun coding 🙃

cc @borda @tchaton

### CODE REVIEW

**Review by akihironitta — APPROVED** [review]

File: requirements/pytorch/strategies.txt [hunk 1]
```diff
@@ -1,7 +1,7 @@
 # NOTE: the upper bound for the package version is only set for CI stability, and it is dropped while installing this package
 #  in case you want to preserve/enforce restrictions on the latest compatible version, add "strict" as an in-line comment
 
-colossalai>=0.1.10
+# colossalai>=0.1.10
```
Comment (carmocca) [inline requirements/pytorch/strategies.txt:4]: I would make an exception and keep this unless we actually start seeing broken master CI from this. If that happens and their devs are unable to help, then we comment the dependency until fixed.

Comment (Borda) [reply requirements/pytorch/strategies.txt:4]: I see broken PR :otter: #15942

Comment (carmocca) [reply requirements/pytorch/strategies.txt:4]:
```suggestion
# colossalai>=0.1.10  # TODO: uncomment when there's a stable version released
```

**Review by carmocca — APPROVED** [review]

**Review by awaelchli — APPROVED** [review]
@Borda Could you let the Colossal AI team know in case you haven't already? :)

**Pull request merged**